tools/tertiary-analysis/decoupler/decoupler_pathway_inference.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="decoupler_pathway_inference" name="Decoupler Pathway Inference" version="1.4.0+galaxy0" profile="20.05" license="MIT">
+    <description>
+        Pathway inference using the Decoupler.
+    </description>
+    <requirements>
+        <requirement type="package" version="1.4.0">decoupler</requirement>
+    </requirements>
+    <command>
+        python '$__tool_directory__/decoupler_pathway_inference.py'
+            --input_file '$input_file'
```

Comment:
input_file is not defined in the `<inputs>`

Reply:
```suggestion
            --input_file '$input_anndata'
```

Reply:
🤦🏾‍♂️